src/components/ErrorBoundary.js

```diff
@@ -1,32 +1,34 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import Alert from 'react-bootstrap/Alert';
 
-class ErrorBoundary extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { hasError: false, error: null };
-  }
+const ErrorBoundary = ({ children, fallback }) => {
+  const { error, handleError } = useErrorHandler();
 
-  static getDerivedStateFromError(error) {
-    // Update state so the next render will show the fallback UI.
-    return { hasError: true, error };
-  }
+  useEffect(() => {
+    try {
 
-  componentDidCatch(error, errorInfo) {
-    console.log('Error: ', error);
-  }
-
-  render() {
-    if (this.state.error) {
-      return (
-        <Alert variant="danger" data-testid="alert-container">
-          <p data-testid="alert-message">Something went wrong...</p>
-        </Alert>
-      );
+    } catch (err) {
+      handleError(err);
     }
+  }, [handleError]);
```

Comment:
So I should ignore this for now because it will be going away in #232?

Reply:
Yes 👍